### ISSUE TITLE
The time is frozen, please help!

### ISSUE THREAD

**Issue by hlouie850** [Rangi42/polishedcrystal#794]
Thank you for this funtastic ROM hack, 8 hours in I realized the time is not moving forward AT ALL! I am playing on 3DS. What should I do to fix this please? Thanks a million!!

**Comment by vulcandth** [comment]
What program did you use to inject the rom? Did you use the `.patch` file that goes with the rom built by 9-bit?

**Comment by hlouie850** [comment]
I use R4 gold pro flash card with an SD memory card. Not sure about the .patch file… where could I find it?

**Comment by vulcandth** [comment]
> I use R4 gold pro flash card with an SD memory card. Not sure about the .patch file… where could I find it?

The `.patch` file is only required if you want to enable the local wireless trading/battling on the 3DS. It isn't a "patch" in the traditional sense, it is just used by the 3DS to help enable the aforementioned capabilities. I only asked, as I was curious if by chance it was causing problems.

What software did you use to put the hack on your R4 gold pro? Did you use something like "New Super Ultimate Injector"? If so, what settings did you choose?

**Comment by vulcandth** [comment]
@FredrIQ mentioned to me that you might be using a different emulator with your R4 cart. Are you using something like "lameboy"? Either if you could just explain what process you used to get the ROM to your cart. :)

**Comment by hlouie850** [comment]
That is correct, lameboy it is!! I basically downloaded the rom file, transferred it across to my SD memory card then R4. Never played online at all.

**Comment by hlouie850** [comment]
> @FredrIQ mentioned to me that you might be using a different emulator with your R4 cart. Are you using something like "lameboy"? Either way, if you could just explain what process you used to get the ROM to your cart that would be helpful. :)

Is there still hope for this software? I am genuinely curious if this issue is fixable. I played the original Crystal version and the day and night cycle is just fine.
